thetis/callback.py

```diff
@@ -299,6 +299,53 @@ def message_str(self, *args):
         return line
 
 
+class TracerScalarConservationCallback(DiagnosticCallback):
+    """Base class for callbacks that check conservation of a scalar quantity"""
+    variable_names = ['integral', 'relative_difference']
+
```

Comment:
I guess you have copied and pasted this from `ScalarConservationCallback`, so it would be good if you changed the docstring.

Reply:
Good spot! I've updated it now